docs/source/stylesheets/components/_fonts.scss

```diff
@@ -53,7 +53,7 @@ p {
     font-size: 16px;
   }
 
-  &+ ul {
+  &+ ul, &+ ol {
```

Comment:
Works for me but let's put in place an automatic linter for later reviews (I always find it non-productive to review coding style/linting manually: this is something that needs to be done automatically or accepted as-is) :)

You can check what we've done for [instantsearch.js](https://github.com/algolia/instantsearch.js/blob/master/.sass-lint.yml).